src/coreclr/src/binder/bindertracing.cpp

```diff
@@ -231,3 +271,178 @@ namespace BinderTracing
         return m_ignoreBind;
     }
 }
+
+namespace BinderTracing
+{
+    ResolutionAttemptedOperation::ResolutionAttemptedOperation(AssemblyName *assemblyName, UINT_PTR binderID, const HRESULT& hr)
```

Comment:
These two overloads feel a bit dangerous - INT_PTR versus UINT_PTR - the two types feel a bit too similar. I would not mind if we had just the one which takes both binderID and managedALC - it makes things explicit.